py-polars/tests/unit/datatypes/test_temporal.py

```diff
@@ -1924,25 +1924,29 @@ def test_replace_timezone_from_to(
     assert result == expected
 
 
-def test_strptime_subseconds_datetime() -> None:
-    with pytest.raises(ValueError, match="not supported"):
-        pl.Series(["2023-02-05T05:10:10.074000"]).str.strptime(
-            pl.Datetime, "%Y-%m-%dT%H:%M:%S.%f"
-        )
-    result = (
-        pl.Series(["2023-02-05T05:10:10.074000"])
-        .str.strptime(pl.Datetime, "%Y-%m-%dT%H:%M:%S%.f")
-        .item()
-    )
-    expected = datetime(2023, 2, 5, 5, 10, 10, 74000)
+@pytest.mark.parametrize(
+    ("format", "expected"),
+    [
+        ("%Y-%m-%dT%H:%M:%S%.f", datetime(2023, 2, 5, 5, 10, 10, 74000)),
+        ("%Y-%m-%dT%H:%M:%S.%f", datetime(2023, 2, 5, 5, 10, 10, 74)),
```

Comment:
this is a perfect example of where it's so unexpected - if you read "`.074000`", you think "oh, that's 74 milliseconds, i.e. 74000 microseconds". But Chrono with `'%f'` will parse it as "74 microseconds"!